docs/data/toolpad/core/components/crud/crud.md

```diff
@@ -0,0 +1,356 @@
+---
+productId: toolpad-core
+title: Crud
+components: Crud, CrudProvider, List, Show, Create, Edit, CrudForm
+---
+
+# Crud
+
+<p class="description">The Crud component provides UIs to interact with data from any data source.</p>
+
+With the `Crud` component and its subcomponents you can easily generate pages where items from your data source can be listed in a table, shown individually in detail, or created and edited with forms. All with minimal configuration from a single data source definition.
+
+## Demo
+
+The `Crud` component automatically generates CRUD pages for your data source, automatically integrated with your routing solution. All that is required is a `dataSource` definition and a `rootPath` for the CRUD pages.
+
+{{"demo": "CrudBasic.js", "height": 600, "iframe": true}}
```

Comment:
Code highlight doesn't seem to be applied for the preview code in case of TS.

Reply:
I know, it doesn't seem like it's ready for components like `<Crud<Note>` but that's valid Typescript, I guess it's an issue with the highlighting...